pystarport/cosmoscli.py

```diff
@@ -337,11 +337,10 @@ def staking_params(self):
         )
 
     def staking_pool(self, bonded=True):
-        return int(
-            json.loads(
-                self.raw("query", "staking", "pool", output="json", node=self.node_rpc)
-            )["bonded_tokens" if bonded else "not_bonded_tokens"]
-        )
+        res = self.raw("query", "staking", "pool", output="json", node=self.node_rpc)
+        res = json.loads(res)
+        res = res.get("pool") if res.get("pool") else res
```

Comment:
```suggestion
        res = res.get("pool") or res
```

Reply:
ref: https://docs.python.org/3/library/stdtypes.html#boolean-operations-and-or-not